node/network/collator-protocol/src/validator_side/mod.rs

```diff
@@ -1537,7 +1549,9 @@ async fn poll_collation_response(
 				CollationFetchResult::Error(Some(COST_NETWORK_ERROR))
 			},
 			Err(RequestError::Canceled(err)) => {
-				gum::debug!(
+				gum::warn_if_frequent!(
```

Comment:
Errors are not the same, so don't we need to have an instance of this per debug message such that we raise to warning only when same type of error is repeated and not mix them ?